### PR TITLE
Add resource integration test

### PR DESCRIPTION
**Pull request by paaige**

### CODE REVIEW

File: qiskit_superstaq/daily_integration_test.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import pytest
 import qiskit
 from applications_superstaq import SuperstaQException
+from applications_superstaq import ResourceEstimate
```
Comment (singular-value) [inline qiskit_superstaq/daily_integration_test.py:8]: nit: alphabetize imports
( or can you do `from applications_superstaq import ResourceEstimate, SuperstaQException`?

File: qiskit_superstaq/daily_integration_test.py [hunk 2]
```diff
@@ -80,6 +81,26 @@ def test_get_balance(provider: qss.SuperstaQProvider) -> None:
...
+
+    assert resource_estimate == ResourceEstimate(1, 1, 2)
+
+    circuit2 = qiskit.QuantumCircuit(3)
+    circuit2.h(1)
+    circuit2.cnot(0, 1)
+    circuit2.cz(1, 0)
+    circuit2.measure(1, 1)
+
+    resource_estimates = provider.resource_estimates([circuit1, circuit2], "neutral_atom_qpu")
```
Comment (singular-value) [inline qiskit_superstaq/daily_integration_test.py:99]: singular (`.resource_estimate(`) for function call right?

**Review by singular-value — APPROVED** [review]
lgtm

**Comment by vtomole** [comment]
@paaige Did you run this to make sure it passes before merging? It seems to fail: https://github.com/SupertechLabs/qiskit-superstaq/runs/7459891669?check_suite_focus=true

**Comment by paaige** [comment]
> @paaige Did you run thison Github Actions to make sure it passes before merging? It seems to fail: https://github.com/SupertechLabs/qiskit-superstaq/runs/7459891669?check_suite_focus=true I tested locally with a server but the list test broke things (and I guess I didn't test after adding that). I'll test via actions going forward. #202

**Pull request merged**